rsciio/delmic/_api.py

```diff
@@ -0,0 +1,77 @@
+import numpy as np
+import h5py as hf
+
+from rsciio._docstrings import FILENAME_DOC, RETURNS_DOC
+
+
+def file_reader(filename, *args, **kwds):
+    """Read a Delmic hdf5 hyperspectral image.
+
+    Parameters
+    ----------
+    %s
+
+    %s
+    """
+    hdf = hf.File(filename, "r")
+
+    Acquisition2 = hdf.get("Acquisition2")
+    Acquisition2_ImageData = Acquisition2.get("ImageData")
+    Acquisition2_ImageData_Image = Acquisition2_ImageData.get("Image")
+
+    Acquisition2_ImageData_DimensionScaleC = Acquisition2_ImageData.get(
+        "DimensionScaleC"
+    )
+    Acquisition2_ImageData_DimensionScaleX = Acquisition2_ImageData.get(
+        "DimensionScaleX"
+    )
+    Acquisition2_ImageData_DimensionScaleY = Acquisition2_ImageData.get(
+        "DimensionScaleY"
+    )
+
+    DATA = Acquisition2_ImageData_Image[:, 0, 0, :, :]
+    data = DATA.transpose()
+    del DATA
+
+    axes = [
+        {
+            "name": "X",
+            "size": data.shape[0],
+            "offset": 0,
+            "scale": float(np.array(Acquisition2_ImageData_DimensionScaleX)),
+            "units": "µm",
+            "navigate": True,
+        },
+        {
+            "name": "Y",
+            "size": data.shape[1],
+            "offset": 0,
+            "scale": float(np.array(Acquisition2_ImageData_DimensionScaleY)),
+            "units": "µm",
+            "navigate": True,
+        },
+        {
+            "name": "Wavelength",
+            "axis": np.array(Acquisition2_ImageData_DimensionScaleC),
+            "units": "nm",
+            "navigate": False,
+        },
+    ]
+
+    metadata = {"signal": {"signal_type": "", "quantity": "Intensity (counts)"}}

```

Comment:
If LumiSpy is installed, `signal_type` should be set to `CL` to provide the CL specific functions.